piker/_daemon.py

```diff
@@ -44,10 +49,34 @@ class Services(BaseModel):
     actor_n: tractor._trionics.ActorNursery
     service_n: trio.Nursery
     debug_mode: bool  # tractor sub-actor debug mode flag
+    ctx_stack: AsyncExitStack
 
     class Config:
         arbitrary_types_allowed = True
 
+    async def open_remote_ctx(
```

Comment:
@guilledk this was the main bit i wanted your review on.

This let's us do deterministic, async setup/teardown from the root `pikerd` nicely.